File: omnipod-dash/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/dash/driver/comm/exceptions/CouldNotEnableNotifications.java
package info.nightscout.androidaps.plugins.pump.omnipod.dash.driver.comm.exceptions;

import info.nightscout.androidaps.plugins.pump.omnipod.dash.driver.comm.CharacteristicType;

public class CouldNotEnableNotifications extends Exception {
    public CouldNotEnableNotifications(CharacteristicType cmd) {
        super(cmd.getValue());
    }
}

File: omnipod-dash/src/main/java/info/nightscout/androidaps/plugins/pump/omnipod/dash/driver/comm/exceptions/CouldNotEnableNotifications.kt
package info.nightscout.androidaps.plugins.pump.omnipod.dash.driver.comm.exceptions

import info.nightscout.androidaps.plugins.pump.omnipod.dash.driver.comm.CharacteristicType

class CouldNotEnableNotifications(cmd: CharacteristicType) : Exception(cmd.value)